File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/rename_scope_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'rename_scope'

include Rename_scope

a = Natural_UP.new
b = Natural_BP.new

raise RuntimeError if a.rtest() != 1

raise RuntimeError if b.rtest() != 1

File: support_tools/swigwin-3.0.12/Examples/test-suite/php/rename_scope_runme.php
<?php

require "tests.php";
require "rename_scope.php";

check::classes(array("rename_scope","Interface_UP","Interface_BP","Natural_UP","Natural_BP","Bucket"));

check::classmethods("Interface_UP",array("__construct","__set","__isset","__get"));
check::classmethods("Interface_BP",array("__construct","__set","__isset","__get"));
check::classmethods("Natural_UP",array("__construct","__set","__isset","__get","rtest"));
check::classmethods("Natural_BP",array("__construct","__set","__isset","__get","rtest"));
check::classparent("Natural_UP","Interface_UP");
check::classparent("Natural_BP","Interface_BP");

check::done();
?>
